### ISSUE TITLE
Remove tsdf_min_camera_movement

### ISSUE THREAD

**Issue by mpowelson** [ros-industrial-attic/yak#20]
This seems like a detail in the data capture not the reconstruction. In other words, I think this should be in yak_ros not yak itself. In fact, it is in both places now. I propose we remove it from yak (in kinfu.cpp) and let the data collection wrapper handle it (yak_ros).

**Comment by mpowelson** [comment]
Along these same lines, update_via_sensor_motion also seems odd. Is there an application for it? I don't know if it currently works.

**Comment by schornakj** [comment]
Agreed on removing the minimum motion threshold from `kinfu.cpp`. The TSDF server should merge whatever images are sent to it, and it should be the client's responsibility to manage when those images are merged.

I tried `update_via_sensor_motion = true` and it doesn't integrate relative to the correct frame. I think it can be safely removed.
